diff -crB /home/quickridge/.vim/ftplugin/latex-suite/compiler.vim ftplugin/latex-suite/compiler.vim
*** /home/quickridge/.vim/ftplugin/latex-suite/compiler.vim	2008-05-04 03:38:08.000000000 -0400
--- ftplugin/latex-suite/compiler.vim	2009-06-15 19:22:45.000000000 -0400
***************
*** 150,155 ****
--- 150,158 ----
  " in the ~/thesis directory. this will then run "latex main.tex" when
  " Tex_RunLaTeX() is called.
  function! Tex_RunLaTeX()
+ 
+ 	" Aaron added this to save file before compiling
+ 	update			
  	call Tex_Debug('+Tex_RunLaTeX, b:fragmentFile = '.exists('b:fragmentFile'), 'comp')
  
  	let dir = expand("%:p:h").'/'
***************
*** 200,205 ****
--- 203,210 ----
  		let i = i + 1
  	endwhile
  	
+ 	redraw!			" Added by me to make sure screen redraws
+ 
  	let s:target = initTarget
  	let s:origwinnum = winnr()
  	call Tex_SetupErrorWindow()
diff -crB /home/quickridge/.vim/ftplugin/latex-suite/folding.vim ftplugin/latex-suite/folding.vim
*** /home/quickridge/.vim/ftplugin/latex-suite/folding.vim	2008-05-04 03:38:08.000000000 -0400
--- ftplugin/latex-suite/folding.vim	2009-06-15 19:22:45.000000000 -0400
***************
*** 284,290 ****
  					" In other words, the pattern is safe, but not exact.
  					call AddSyntaxFoldItem('^\s*\\'.s.'{[^{}]*$','^[^}]*}',0,0)
  				else
- 					call AddSyntaxFoldItem('^\s*\\begin{'.s,'^\s*\\end{'.s,0,0)
  					" call AddSyntaxFoldItem('^\s*\\begin{'.s,'^\s*\\end{'.s,0,0)
  					call AddSyntaxFoldItem('^\s*\\begin{'.s,'\(^\|\s\)\s*\\end{'.s,0,0)
  				endif
--- 284,289 ----
