*** /home/quickridge/.vim/plugin/imaps.vim	2009-06-15 23:42:16.000000000 -0400
--- plugin/imaps.vim	2009-06-15 23:41:26.000000000 -0400
***************
*** 654,660 ****
  " }}}
  " s:RemoveLastHistoryItem: removes last search item from search history {{{
  " Description: Execute this string to clean up the search history.
! let s:RemoveLastHistoryItem = ':call histdel("/", -1)|let @/=g:Tex_LastSearchPattern'
  
  " }}}
  " s:Hash: Return a version of a string that can be used as part of a variable" {{{
--- 654,660 ----
  " }}}
  " s:RemoveLastHistoryItem: removes last search item from search history {{{
  " Description: Execute this string to clean up the search history.
! let s:RemoveLastHistoryItem = ':call histdel("/", -1)|let @/=g:Tex_LastSearchPattern|'
  
  " }}}
  " s:Hash: Return a version of a string that can be used as part of a variable" {{{
